scripts/slave_setting/sshfs/resync_shared_dirs.sh

```diff
@@ -10,6 +10,14 @@ dirs=("repository" "repo" "LOGS")
 echo "List of sshfs mounts:"
 mount | grep sshfs
 
+LS_ERROR=$(ls LOGS)
```

Comment:
Please add 2>&1 to command
And move it inside the for dir_to_resync in "${dirs[@]}"